internal/witness/cmd/witness/internal/witness/witness.go

```diff
@@ -51,11 +57,23 @@ func New(wo Opts) (*Witness, error) {
 	// Create the sths table if needed.
 	_, err := wo.DB.Exec(`CREATE TABLE IF NOT EXISTS sths (logID BLOB PRIMARY KEY, sth BLOB)`)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to create table: %v", err)
+	}
+	// Parse the PEM-encoded secret key.
+	p, _ := pem.Decode([]byte(wo.PrivKey))
+	if p == nil {
+		return nil, fmt.Errorf("no PEM block found in %s", wo.PrivKey)
```

Comment:
I feel a bit conflicted about this - returning the secret key string here where it'll likely end up being logged seems dangerous... but might help to figure out what's wrong :/

Reply:
As with many things I've just copied this from elsewhere in the repo... happy to take suggestions for alternative approaches!  I guess the most direct one is just saying that the formatting is wrong without printing the secret key?

Reply:
Yeah, that seems like the best compromise.

Reply:
Okay, have gone with that.